src/Akeneo/Platform/Bundle/ImportExportBundle/front/src/components/StorageConfigurator/MicrosoftAzureStorageConfigurator.test.tsx

```diff
@@ -0,0 +1,273 @@
+    ),
+    'name'
+  );
+
+  expect(onStorageChange).toHaveBeenLastCalledWith({
+    ...storage,
+    container_name: 'container_name',
+  });
+});
+
+test('it throws an exception when passing a non microsoft azure storage', async () => {
+  const mockedConsole = jest.spyOn(console, 'error').mockImplementation();
+
+  const storage: LocalStorage = {
+    type: 'local',
+    file_path: '/tmp/file.xlsx',
+  };
+
+  expect(() =>
+    renderWithProviders(
+      <MicrosoftAzureStorageConfigurator
+        storage={storage}
+        fileExtension="xlsx"
+        validationErrors={[]}
+        onStorageChange={jest.fn()}
+      />
+    )
+  ).toThrowError('Invalid storage type "local" for microsoft azure storage configurator');
+
+  mockedConsole.mockRestore();
+});
+
+test('it displays validation errors', async () => {
+  const storage: MicrosoftAzureStorage = {
+    type: 'microsoft_azure',
+    file_path: '/tmp/file.xlsx',
+    connection_string: 'connection_string',
+    container_name: 'container_name',
+  };
+
+  const validationErrors: ValidationError[] = [
+    {
+      messageTemplate: 'error.key.a_file_path_error',
+      invalidValue: '',
+      message: 'this is a file_path error',
+      parameters: {},
+      propertyPath: '[file_path]',
+    },
+    {
+      messageTemplate: 'error.key.a_connection_string_error',
+      invalidValue: '',
+      message: 'this is a connection string error',
+      parameters: {},
+      propertyPath: '[connection_string]',
+    },
+    {
+      messageTemplate: 'error.key.a_container_name_error',
+      invalidValue: '',
+      message: 'this is a container name error',
+      parameters: {},
+      propertyPath: '[container_name]',
+    },
+  ];
+
+  await act(async () => {
+    renderWithProviders(
+      <MicrosoftAzureStorageConfigurator
+        storage={storage}
+        fileExtension="xlsx"
+        validationErrors={validationErrors}
+        onStorageChange={jest.fn()}
+      />
+    );
+  });
+
+  expect(screen.getByText('error.key.a_file_path_error')).toBeInTheDocument();
+  expect(screen.getByText('error.key.a_connection_string_error')).toBeInTheDocument();
+  expect(screen.getByText('error.key.a_container_name_error')).toBeInTheDocument();
+});
+
+test('it can check connection', async () => {
+  const storage: MicrosoftAzureStorage = {
+    type: 'microsoft_azure',
+    file_path: '/tmp/file.xlsx',
+    connection_string: 'connection_string',
+    container_name: 'container_name',
+  };
+
+  const onStorageChange = jest.fn();
+
+  renderWithProviders(
+    <MicrosoftAzureStorageConfigurator
+      storage={storage}
+      fileExtension="xlsx"
+      validationErrors={[]}
+      onStorageChange={onStorageChange}
+    />
+  );
+
+  const checkButton = screen.getByText('pim_import_export.form.job_instance.connection_checker.label');
+  await act(async () => {
+    userEvent.click(checkButton);
+  });
+
```

Comment:
In order to check that there is no error message
```suggestion 
expect(screen.getByText('pim_import_export.form.job_instance.connection_checker.exception')).not.toBeInTheDocument();
```

May assert the success message also